src/hooks/test/use-ordered-rows-test.js

```diff
@@ -0,0 +1,129 @@
+import { mount } from 'enzyme';
+import { useState } from 'preact/hooks';
+
+import { useOrderedRows } from '../use-ordered-rows';
+
+const starWarsCharacters = [
+  { name: 'Luck Skywalker', age: 20 },
+  { name: 'Princess Leia Organa', age: 20 },
+  { name: 'Han Solo', age: 25 },
+];
+
+describe('useOrderedRows', () => {
+  function FakeComponent() {
+    const [order, setOrder] = useState();
+    const orderedRows = useOrderedRows(starWarsCharacters, order);
+
+    return (
+      <div>
+        {orderedRows.map((character, index) => (
+          <div key={`${character.name}${index}`}>
+            <span data-testid={`name-${index}`}>{character.name}</span>
+            <span data-testid={`age-${index}`}>{character.age}</span>
+          </div>
+        ))}
+        <button
+          data-testid="button-order-by-name-asc"
+          onClick={() => setOrder({ field: 'name', direction: 'ascending' })}
+        >
+          By name ASC
+        </button>
+        <button
+          data-testid="button-order-by-name-desc"
+          onClick={() => setOrder({ field: 'name', direction: 'descending' })}
+        >
+          By name DES
+        </button>
+        <button
+          data-testid="button-order-by-age-asc"
+          onClick={() => setOrder({ field: 'age', direction: 'ascending' })}
+        >
+          By age ASC
+        </button>
+        <button
+          data-testid="button-order-by-age-desc"
+          onClick={() => setOrder({ field: 'age', direction: 'descending' })}
+        >
+          By age DES
+        </button>
+        <button
+          data-testid="button-reset-order"
+          onClick={() => setOrder(undefined)}
+        >
+          Reset order
+        </button>
+      </div>
+    );
+  }
+
+  function createComponent() {
+    return mount(<FakeComponent />);
+  }
+
+  function assertDefaultOrder(wrapper) {
+    assertOrder(wrapper, starWarsCharacters);
+  }
+
+  function assertOrder(wrapper, rows) {
+    rows.forEach((character, index) => {
+      assert.equal(
+        character.name,
+        wrapper.find(`[data-testid="name-${index}"]`).text(),
+      );
```

Comment:
Assert syntax should be `assert.equal(actual, expected)`. See [the docs](https://www.chaijs.com/api/assert/#method_equal). Therefore the parameters need to be swapped. Naming `rows` to `expectedRows` would make it more immediately obvious that `rows` refers to the expected values.